beacon-chain/db/filesystem/blob.go

```diff
@@ -1,42 +1,29 @@
 package filesystem
 
 import (
-	"context"
 	"fmt"
 	"math"
-	"os"
 	"path"
-	"strconv"
-	"strings"
 	"time"
 
-	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/prysm/v5/beacon-chain/verification"
 	fieldparams "github.com/prysmaticlabs/prysm/v5/config/fieldparams"
 	"github.com/prysmaticlabs/prysm/v5/consensus-types/blocks"
 	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
-	"github.com/prysmaticlabs/prysm/v5/encoding/bytesutil"
 	"github.com/prysmaticlabs/prysm/v5/io/file"
-	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
 	"github.com/prysmaticlabs/prysm/v5/runtime/logging"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/afero"
 )
 
+const directoryPermissions = 0700
```

Comment:
I feel like we have this set differently for different files in the code base, should we standardize the permissions somewhere? ( need to double check)

Reply:
Looks like we've got [this](https://github.com/prysmaticlabs/prysm/blob/develop/config/params/io_config.go#L18) dusty old helper, I can use it.

Reply:
updated